### PR TITLE
Add basic start page

### PR DESCRIPTION
**Pull request by georgeschena**
[Jira ticket](https://cddodatamarketplace.atlassian.net/jira/software/projects/GDX/boards/1/backlog?cloudId=79ba83b4-8d57-460d-9968-f5a8eb5bb565&selectedIssue=GDX-192&text=dt)

Add a basic start page to the app.

Usually this would be hosted on GOV.UK but as this is a prototype/MVP we will host our own start page to replicate the experience for out user testing. Still needs content added.

### Screenshot
Screenshot 2023-02-03 at 11 40 18

### CODE REVIEW

**Review by DarrylMultisided — APPROVED** [review]
This should be merged before Jason's PR as it moves the ruleset controller on a bit.

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,30 +1,17 @@
...
-
-      <%= submit_tag("Continue", class: "govuk-button", data: { module: "govuk-button" }) %>
-    <% end %>
-
+    <p>
+      <a class="gem-c-button govuk-button govuk-button--start" role="button" data-module="govuk-button" draggable="false" href="/ruleset" data-govuk-button-module-started="true">
+        Start now
+        <svg class="govuk-button__start-icon govuk-!-display-none-print" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" focusable="false" aria-hidden="true">
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+        </svg>
```
Comment (RobNicholsGDS) [inline app/views/pages/home.html.erb:11]: There is [a govuk component](https://govuk-components.netlify.app/components/start-button/) that will reduce this code down (lines 7-12) to:
```ruby
<%= govuk_start_button(text: "Start now", href: "/ruleset") %>
```

File: app/views/ruleset/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      <div class="govuk-form-group" name="which-linter">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h1 class="govuk-fieldset__heading">
+              Which ruleset would you like to test your OpenAPI document against?
+            </h1>
+          </legend>
+          <div class="govuk-radios" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+              <%= text_field_tag("which_ruleset", "security_ruleset", class: "govuk-radios__input", type: "radio", name: "which_ruleset" ) %>
```
Comment (RobNicholsGDS) [inline app/views/ruleset/index.html.erb:15]: Why is that a text field? Shouldn't it be a radio?

File: config/routes.rb [hunk 3]
```diff
@@ -2,11 +2,7 @@
   root to: "pages#home"
 
   # post '/linter', to: "linter#new"
-  resources :ruleset, only: [:new, :create]
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :ruleset, only: %i[new create index]
```
Comment (RobNicholsGDS) [inline config/routes.rb:5]: Using a singular with `resources` causes some strangeness with the index and create path helpers. If this line is changed to:
```ruby
resources :rulesets, only: %i[new create index]
```
The path helper will be:
- new `new_ruleset_path`
- create `rules_sets_path`
- index `rules_sets_path`

**Pull request merged**